zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -487,21 +492,22 @@ mod tests {
                          END
                        ) AS memo_count
                 FROM sent_notes
+                JOIN transactions
+                     ON transactions.id_tx = sent_notes.tx
                 LEFT JOIN sapling_received_notes
                           ON (sent_notes.tx, sent_notes.output_pool, sent_notes.output_index) =
                              (sapling_received_notes.tx, 2, sapling_received_notes.output_index)
                 WHERE  sapling_received_notes.is_change IS NULL
                    OR  sapling_received_notes.is_change = 0
```

Comment:
```suggestion
                WHERE COALESCE(sapling_received_notes.is_change, 0) = 0
```
Non-blocking, just a style nit.